contracts/Challenge/FutarchyChallenge.sol

```diff
@@ -19,8 +19,9 @@ contract  FutarchyChallenge is ChallengeInterface {
   address public challenger;         // the address of the challenger
   address public listingOwner;       // the address of the listingOwner
   bool public isStarted;             // true if challenger has executed start()
+  bool public marketsAreClosed;      // true if futarchy markets are closed
   uint public stakeAmount;           // number of tokens to stake for either party during challenge
-  uint public tradingPeriod;         // duration for open trading on scalar prediction markets
+  uint public tradingPeriod;         // duration for open trading before futarchy resolution
```

Comment:
Does "futarchy resolution" mean when the decision is made?

Reply:
Cool, that makes sense.  I think I see the term `futarchy` encompassing more than the decision being made, and including the financial reward or penalty to the market participants, but debating that is probably out of scope here :)

Reply:
yah if there's a better term like `futarchy decision resolution` or something that you like better, lmk

Reply:
I like `futarchy decision resolution` better.  Cuts out any ambiguity in my mind.